lte/gateway/python/load_tests/common.py

```diff
@@ -84,7 +75,7 @@ def benchmark_grpc_request(
     output_file: str,
     num_reqs: int,
     address: str,
-    import_path: str = None,
+    import_path: str,
```

Comment:
It turns out, that the import path is not actually optional, but always given by the caller.
I.e. the logic to "intelligently" determine some protobuf path can be removed.